### PR TITLE
Update version and fix typos/syntax found in #28

### PR DESCRIPTION
**Pull request by gtauzin**
-->#### What does this implement/fix? Explain your changes.
- Fixes typos and syntax as per @ulupo comments in #28.
- Updates version to 0.2.2

### CODE REVIEW

File: pyflagser/flagio.py [hunk 1]
```diff
@@ -65,11 +65,11 @@ def saveflag(fname, flag_matrix):
         graph. Diagonal elements are vertex weights. The way zero values are
         handled depends on the format of the matrix. If the matrix is a dense
         `np.ndarray`, all zeros are explicitly accounted for and denote
-        zero-weight edges, i.e., edges appeaing at filtration value zero.
+        zero-weight edges, i.e., edges appearing at filtration value zero.
         If the matrix is a sparse `scipy.sparse` matrix, zeros on the diagonal
         and off-diagonal zeros assigned directly are treated explicitly. Off-
         diagonal zeros that have not been assigned directly are treated
-        implicitly, i.e., corresponds to an abscence of edge.
+        implicitly, i.e., correspond to an absent of edge.
```
Comment (ulupo) [inline pyflagser/flagio.py:72]: Remove "of" from "absent of"

File: pyflagser/flagser.py [hunk 2]
```diff
@@ -20,11 +20,11 @@ def flagser(flag_matrix, min_dimension=0, max_dimension=np.inf, directed=True,
         graph. Diagonal elements are vertex weights. The way zero values are
         handled depends on the format of the matrix. If the matrix is a dense
         `np.ndarray`, all zeros are explicitly accounted for and denote
-        zero-weight edges, i.e., edges appeaing at filtration value zero.
+        zero-weight edges, i.e., edges appearing at filtration value zero.
         If the matrix is a sparse `scipy.sparse` matrix, zeros on the diagonal
         and off-diagonal zeros assigned directly are treated explicitly. Off-
         diagonal zeros that have not been assigned directly are treated
-        implicitly, i.e., corresponds to an abscence of edge.
+        implicitly, i.e., correspond to an absent of edge.
```
Comment (ulupo) [inline pyflagser/flagser.py:27]: Ditto as above.

File: pyflagser/flagio.py [hunk 3]
```diff
@@ -26,13 +26,13 @@ def loadflag(fname, fmt='csr', dtype=None):
...
         Matrix representation of a directed/undirected weighted/unweighted
         graph. Diagonal elements are vertex weights. The way zero values are
-        handled depends on the format of the matrix. If the matrix is a dense
-        `np.ndarray`, all zeros are explicitly accounted for and denote
-        zero-weight edges, i.e., edges appeaing at filtration value zero.
-        If the matrix is a sparse `scipy.sparse` matrix, zeros on the diagonal
-        and off-diagonal zeros assigned directly are treated explicitly. Off-
-        diagonal zeros that have not been assigned directly are treated
-        implicitly, i.e., corresponds to an abscence of edge.
+        handled depends on `fmt`. If `fmt` is `'dense'`, all zeros are
```
Comment (ulupo) [inline pyflagser/flagio.py:29]: I think we need to use double ticks ( `` ) where we want monospace fonts, e.g. in the case of ``'dense'`` and ``scipy.sparse``.  Single ticks display as italics in the docs.

**Comment by gtauzin** [comment]
@ulupo Thanks!

**Pull request merged**